sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -53,6 +53,15 @@ now raise more concrete exception other than `MessageSettleFailed` and `ServiceB
   - Added support for Python 3.9.
   - Fixed bug where amqp message `footer` and `delivery_annotation` were not encoded into the outgoing payload.
 
+**Breaking Changes**
+
+* `amqp_message` has been renamed to `amqp_annotated_message` for cross-sdk consistency.
```

Comment:
Do you think we should also rename the class `AmqpMessage` to `AmqpAnnotatedMessage` in full consistency with other languages?

Reply:
What hesitance I might have via terseness is overridden by alignment with the typename since a user will never initialize.  Will do.++